kong/plugins/influxdb-log/schema.lua

```diff
@@ -0,0 +1,9 @@
+local utils = require "kong.tools.utils"
+local http_log_schema = require "kong.plugins.http-log.schema"
+
+local influxdb_log_schema = utils.deep_copy(http_log_schema)
+
+influxdb_log_schema.fields.method = { default = "POST", enum = { "POST" } }
```

Comment:
I can see that, but if only POST is allowed, what's the use of making it configurable?

Or is it that InfluxDB allows other methods, but they've just not been implemented in the plugin?